### PR TITLE
chore: improved accuracy of round() by more consistent use of tolerance. The latter is adjusted to be lower. JsonQueryProcessor now prints doubles up to scale of 12.

### PR DESCRIPTION
**Pull request by bluestreak01**

### CODE REVIEW

File: core/src/main/java/io/questdb/cutlass/pgwire/BindVariableSetter.java [hunk 1]
```diff
@@ -27,6 +27,6 @@
 import io.questdb.griffin.SqlException;
 
 @FunctionalInterface
-interface BindVariableSetter {
+public interface BindVariableSetter {
```
Comment (mpsq) [inline core/src/main/java/io/questdb/cutlass/pgwire/BindVariableSetter.java:30]: Do you think this change could fix the issue we are having with bound parameters and PGWire?

File: core/src/test/java/io/questdb/griffin/KeyedVectorAggregateTest.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ ******************************************************************************/
+
+package io.questdb.griffin;
```
Comment (mpsq) [inline core/src/test/java/io/questdb/griffin/KeyedVectorAggregateTest.java:25]: This change seems to be unrelated, was it added accidentally or it is on purpose?

Comment (mpsq) [reply core/src/test/java/io/questdb/griffin/KeyedVectorAggregateTest.java:25]: Same with std.Hash file

Comment (bluestreak01) [reply core/src/test/java/io/questdb/griffin/KeyedVectorAggregateTest.java:25]: accidentally. I stashed other changes but forgot this file in GIT.
